src/test/java/xyz/vegaone/easytracking/project/ProjectServiceTest.java

```diff
@@ -61,11 +63,8 @@ public void getProject(){
     @Test
     public void deleteTask(){
         //given
-        Project project = new Project();
-        project.setDescription(PROJECT_DESCRIPTION);
-        project.setName(PROJECT_NAME);
 
-        Project savedProject = projectService.createProject(project);
+        Project savedProject = projectService.createProject(createNewProject());
 
         //when
```

Comment:
minor: could you please format the code so that it will look something like: 
```java
//given
some code

//when
some code

//then
some code
```